engine/chain/internal/bhs/healthcheck_test.go

```diff
@@ -0,0 +1,51 @@
+package bhs_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/chain"
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/errors"
+	"github.com/bitcoin-sv/spv-wallet/engine/chain/models"
+	"github.com/bitcoin-sv/spv-wallet/engine/tester"
+	"github.com/stretchr/testify/require"
+)
+
+/**
+NOTE: switch httpClient to resty.New() tu call actual ARC server
+*/
+
+func TestHealthcheckBHS(t *testing.T) {
+	t.Run("BHS Healthcheck success", func(t *testing.T) {
```

Comment:
Same here as my comment about healthcheck

Reply:
Resolved according to discussion above ☝️.